assets/sass/local.sass

```diff
@@ -5,3 +5,9 @@ table th, table td, table .form-control
   font-size: 16px
   @media print
     font-size: 8px
+
+.govuk-table__cell-no-border
```

Comment:
not sure what `locals.sass` is ... but from what it sounds like - this wont be applied in dev environment?

Reply:
Was already there - its just imported as part of the application.sass - think it means local to the application